src/pages/Calculator/Steps/ReviewMix/StripedLabels.js

```diff
@@ -17,7 +17,7 @@ const StripedLabels = ({ seed, data, labels }) => {
             sx={{ backgroundColor: !(i % 2) && "#e3e5d3" }}
             xs={12}
           >
-            <Grid item sx={{ textAlign: "justify" }} xs={10}>
+            <Grid item sx={{ textAlign: "justify" }} xs={10} pl={1}>
```

Comment:
we don't want padding or margin in grids anymore right?

Reply:
Only margins in Grid would cause offset problems, padding is fine.